ci-operator/config/openshift/velero-plugin-for-csi/openshift-velero-plugin-for-csi-main.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  base:
+    name: "4.13"
+    namespace: ocp
+    tag: base
+  os:
+    name: centos
+    namespace: openshift
+    tag: "7"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- dockerfile_path: Dockerfile.ubi
+  from: base
+  to: velero-plugin-for-csi
+releases:
```

Comment:
From my understanding you're claiming a release that you don't use. Is it intentional?

Reply:
This is intentional for now.

Reply:
@danilo-gemoli Question: I have unit test configured in this file. Should I expect that running in the checks below?